app/src/main/java/app/revanced/manager/ui/screen/PatchesSelectorScreen.kt

```diff
@@ -192,10 +191,9 @@ fun PatchesSelectorScreen(
     if (showSelectionWarning) {
         SelectionWarningDialog(onDismiss = { showSelectionWarning = false })
     }
-    vm.pendingUniversalPatchAction?.let {
+    viewModel.pendingUniversalPatchAction?.let {
```

Comment:
Storing a lambda is probably unnecessary now since continuing is no longer possible.